akka-remote/src/main/java/akka/remote/artery/compress/CountMinSketch.java

```diff
@@ -0,0 +1,338 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
this is not our ordinary header, it was copied from clearspring but since it contains no copyright information, only ordinary license info (which is the same as ours) I see no reason to include it here. The later comment about derived work from clearspring should be enough